try:
    print(x)
except:
    print("An Exception occur")    